full/Dockerfile

```
@@ -233,4 +233,10 @@ RUN for i in $(ls /var/lib/apt/dazzle-marks/*.gpg); do apt-key add "$i"; done
 # copy tests to enable the self-test of this image
 COPY tests /var/lib/dazzle/tests
 
+# share env see https://github.com/gitpod-io/workspace-images/issues/472
+USER gitpod
+RUN echo "PATH="${PATH}"" > /tmp/env.txt
+USER root
+RUN cat /tmp/env.txt >> /etc/environment && npx envinfo
```

Many thanks for implementing this fix! This is awesome.

Nit: I'd prefer not switching to `USER root` (it's not necessary), and not running `npx envinfo` in our build if possible.

```suggestion
RUN echo "PATH="${PATH}"" | sudo tee -a /etc/environment
```